cubed/array_api/statistical_functions.py

```diff
@@ -152,3 +153,12 @@ def sum(x, /, *, axis=None, dtype=None, keepdims=False):
         keepdims=keepdims,
         extra_func_kwargs=extra_func_kwargs,
     )
+
+
+def var(x, /, *, axis=None, keepdims=False):
```

Comment:
This doesn't match the signature in the [specification](https://data-apis.org/array-api/latest/API_specification/statistical_functions.html) since it's missing a `correction` parameter. Similarly for `std`.